docs/P1.ipynb

```diff
@@ -0,0 +1,593 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 1: Getting Started with Basic Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### **Purpose**\n",
+    "\n",
+    "\n",
+    "The purpose of this demo is to showcase the utility of some common data exploration methods using the OpenDP Context API with the Polars Integration. The notebook will focus on statistical releases in the *trusted curator* setting.\n",
+    "\n",
+    "Throughout this notebook, we will make statistical queries to data with and without privacy-preserving mechanisms. As we compare query results side-by-side, we show that conclusions about the data are similar in both settings: without privacy-preserving mechanism, and with differential privacy mechanism. Let's get started! \n",
```

Comment:
Would it be possible to have DP confidence intervals, perhaps with whisker plots? Otherwise I'm wondering if we just happened to get lucky.

Reply:
for the quantiles?